docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+
+<AposCodeBlock>
+::: v-pre
+
+```django
+{% import 'fragments/navigation.html' as navigation %}
+{% import 'fragments/footer.html' as footer %}
+
+{# Automatically extends the right outer layout and also handles AJAX siutations #}
+{% extends data.outerLayout %}
+
+{% set title = data.piece.title or data.page.title %}
+{% block title %}
+  {{ title }}
+  {% if not title %}
+    {{ apos.log('Looks like you forgot to override the title block in a template that does not have access to an Apostrophe page or piece.') }}
+  {% endif %}
+{% endblock %}
+
+{% block extraHead %}
+  <!-- Font Awesome icons (free version)-->
+  <script src="https://use.fontawesome.com/releases/v6.1.0/js/all.js" crossorigin="anonymous"></script>
+  <!-- Google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css" />
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800" rel="stylesheet" type="text/css" />
+{% endblock %}
+
+{% block beforeMain %}
+  {% render navigation.navigationArea() %}
+{% endblock %}
+
+<main class="mb-4">
+{% block main %}
+  {#
+    Usually, your page templates in the @apostrophecms/pages module will override
+    this block. It is safe to assume this is where your page-specific content
+    should go.
+  #}
+{% endblock %}
+</main>
+
+{% block afterMain %}
+  {% render footer.footerArea() %}
+{% endblock %}
+
+```
+:::
+<template v-slot:caption>
+views/layout.html
+</template>
+</AposCodeBlock>
+
+Note the use of `navigationArea()` and `footerArea()` in the `render` calls. If you look at the fragment files, you will see that the fragment blocks use these names. This is to allow for a single fragment file to have multiple fragments. We are using these names just to be a little clearer in our calls, but they could just be named `navigation()` and `footer()`.
+
+## Creating a default page
+
+Now all of our pages will have our navigation and footer areas, but we need to add our header and all of the body content. We could make a separate file for each page of our site, but it makes sense to have a default page that will be used for the non-blog pages. We can use the existing `modules/default-page`.
+
+We need to make two major modifications to the existing `default-page`. First, we need to import and display our header fragment. In this case, we want to *add* the header into the `beforeMain` block. In order to accomplish this we need to add the 
+::: v-pre
```

Comment:
Does this need the `v-pre`? Makes a bit of a weird break on the rendered page.

Reply:
Yeah, we use VuePress. This allows you to do a lot of templating things with Vue components, but it also means that you can't use double mustaches, even inside of backticks - Grrr!

Reply:
Booo 👻